src/api/spec/models/project_spec.rb

```diff
@@ -33,7 +33,7 @@
             </project>
           EOF
         )
-        project.update_repositories(xml_hash, force = false)
+        project.update_repositories(xml_hash, false)
```

Comment:
I don't like the idea to make the code more complex using a hash just to improve readability. If you don't know what the `force` means you can just look the function definition and there the variable that is called `force`.